### PR TITLE
added cc and bcc arguments to mail.smtp command

### PR DESCRIPTION
**Pull request by przegwar**

### CODE REVIEW

File: G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs [hunk 1]
```diff
@@ -68,12 +75,20 @@ public void Execute(Arguments arguments)
             client.Credentials = new System.Net.NetworkCredential(arguments.Login.Value, arguments.Password.Value);
             client.Timeout = (int)arguments.Timeout.Value.TotalMilliseconds;
 
-            string from = arguments.From.Value;
-            string to = arguments.To.Value;
-            string subject = arguments.Subject.Value;
-            string body = arguments.Body.Value;
+            var from = arguments.From.Value;
+            var to = arguments.To.Value;
+            var cc = arguments.Cc.Value.Split(',').ToList();
```
Comment (DanielWilczek) [inline G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs]: we also can use ';' to split email addresses

File: G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs [hunk 2]
```diff
@@ -68,12 +75,20 @@ public void Execute(Arguments arguments)
...
+            var cc = arguments.Cc.Value.Split(',').ToList(); 
+            var bcc = arguments.Bcc.Value.Split(',').ToList();
+            var subject = arguments.Subject.Value;
+            var body = arguments.Body.Value;
 
             MailMessage mm = new MailMessage(from, to, subject, body);
+            cc.Where(a => !string.IsNullOrEmpty(a))
+                .ToList()
+                .ForEach(a =>mm.CC.Add(a));
+            bcc.Where(a => !string.IsNullOrEmpty(a))
```
Comment (DanielWilczek) [inline G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs]: bcc.Where(a => !string.IsNullOrEmpty(a.Trim()))

File: G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs [hunk 3]
```diff
@@ -68,12 +75,20 @@ public void Execute(Arguments arguments)
...
+                .ToList()
+                .ForEach(a => mm.Bcc.Add(a));
```
Comment (DanielWilczek) [inline G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs]: bcc.Where(a => !string.IsNullOrEmpty(a.Trim()))

File: G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs (hunk 1 above)
Comment (przegwar) [reply G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs]: The decision was made that we will only use ';' to split the addresses

File: G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs [hunk 4]
```diff
@@ -37,6 +38,12 @@ public class Arguments : CommandArguments
             [Argument(Required = true, Tooltip = "Receiver's email address")]
             public TextStructure To { get; set; }
 
+            [Argument(Required = true, Tooltip = "Carbon copy")]
```
Comment (pwitusik) [inline G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs]: add informative message to the tooltip how many email addresses can be passed

File: G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs [hunk 5]
```diff
@@ -37,6 +38,12 @@ public class Arguments : CommandArguments
...
+            public TextStructure Cc { get; set; } = new TextStructure(string.Empty);
+
+            [Argument(Required = true, Tooltip = "Blind carbon copy")]
```
Comment (pwitusik) [inline G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs]: add informative message to the tooltip how many email addresses can be passed

**Pull request merged**